### ISSUE TITLE
MUSIC FILES

### ISSUE THREAD

**Issue by projecterik** [AzuraCast/AzuraCast#4065]

**Comment by projecterik** [comment]
Also ERRORS.
See screenshot.
![thumbnail_IMG-20210423-WA0000](https://user-images.githubusercontent.com/80392954/115932736-cf49db00-a48d-11eb-9e24-5fc9efb85f0b.jpg)

**Comment by projecterik** [comment]
It is not working in the menu

**Comment by Vaalyn** [comment]
As I said in the previous issue, you need to follow the issue template.

If you are having trouble understanding what is written in the template please use something like Google Translate to translate the text.

I understand that you are encountering a problem and are frustrated by that but we will only provide support if we get the information that we ask for in the issue template. We do not want to invest a lot of time on asking basic questions that are asked in the template and that we need to start debugging things.

In the template you will see questions that look like this:

```
**Using Docker installation method**
<!-- Yes / No -->
```

You have to edit this to match your setup / write what that part is asking for, like this:

```
**Using Docker installation method**
Yes
```

I will re-open this issue when you have filled out the issue template.

**Comment by projecterik** [comment]
Please open up team vieuwer and login to AzuraCast.
It is working poor

**Comment by Vaalyn** [comment]
We are not providing hands-on support.

Fill out the issue template and provide us with the information we ask for.
Then I will re-open the issue.

**Comment by projecterik** [comment]
Dont understand what you mean with template use proper english

**Comment by projecterik** [comment]
were can i find the template?

**Comment by Vaalyn** [comment]
Visit https://github.com/AzuraCast/AzuraCast/issues/new?assignees=&labels=&template=bug_report.md

Read the text in the text box and answer the questions that are written there.

Look at other issues on our[ issue list](https://github.com/AzuraCast/AzuraCast/issues) as reference on how this should look like.

I can't explain this in simpler terms.

**Comment by projecterik** [comment]
Again were can i find this to solve this.
Dont keep sending links that dont HELP me

**Comment by Vaalyn** [comment]
Have you even tried opening the page I linked and actually read what's written in the text box??

**Comment by projecterik** [comment]
PLEASE AGAIN TRY TO KEEP IT SIMPLE I AM NOT A COMPUTER NERD
I AM A DJ!
\
\SO HELP ME AND KEEP THIS CONVERSTAION SIMPLE>AND DONT TALK IN COMPURTER TALK!

**Comment by projecterik** [comment]
MAny thanks FOR AZURACAST for helping me.Free of cost is not the answer.
Better buy somting then you have Guarantee

**Comment by projecterik** [comment]
IT is stil not working.
It keeps loading
Why dont you help me.
You are the creator of this.

**Comment by Vaalyn** [comment]
I am not helping you since you don't try to spent the minimum amount of effort on understanding what we are asking of you which is providing us with basic information.

We can't make it any simpler than telling you to answer the questions written on [this page](https://github.com/AzuraCast/AzuraCast/issues/new?assignees=&labels=&template=bug_report.md).

There is nothing in what I have written that is "special" which only "computer nerds" can understand.

If you do not want to read the text on the page that I linked to and don't want to answer the questions from that text and you want hands-on support where someone directly looks at your server in order to figure out what problem you are having then as you said you really should use something where you pay for that type of support.

**Comment by projecterik** [comment]
I have read al you sad.
But ypu are not reading my problems

So again: i pres music files in azuracast menu but it wont show my music files.
Dificult to understand?

**Comment by projecterik** [comment]
And ask money for it as it makes it better in futere times
Donateing is not the solution.
As no one does donations for al the work you do. Better ask money for it.
Its an idea keep in mind

**Comment by SlvrEagle23** [comment]
@projecterik We understand the problem, you just haven't told us any information that would let us actually help and fix the issue. Like, what version are you on? Which release channel (stable or rolling)? Docker or no?

Mainly we need to know what is causing your browser to do that; it's a kind of complicated process that involves opening the inspector panel in your browser and showing us a response error.

If you don't want to do that kind of stuff (because it's too "computer nerdy") you can just try to update again to the latest version...that might fix it. More often than not it does.

But more than anything, be nice. You're dealing with volunteers giving away free software. Remember the human you're talking to.

**Comment by projecterik** [comment]
That is the problem.
It is just gone to  an new update.
So it is running on the new version.
And now i have this issue.
Before update it was al good

**Comment by projecterik** [comment]
Test your updates before releasing it.
That makes the system works better.
Now i have the problems with your software.,
Always after updates i have those issues.
Do testing first before updating your software

**Comment by SlvrEagle23** [comment]
@projecterik We do.

You are not being kind or patient, and your behavior is very inappropriate for dealing with volunteers writing free software.

**Comment by projecterik** [comment]
Do you have a job for me?
MAybe i can help.
But for now i have problems with your software.
And you want me to be friendly?
I have more then 1000 GB of music.
And i dont have acces now to it

**Comment by SlvrEagle23** [comment]
@projecterik No, we don't want you to be friendly, we **require** you to be friendly. We're under no obligation to provide _any_ support for our software, and yet we spend a LOT of time supporting users, but we absolutely will not support someone who is rude to us.

If you can get an error message (from the inspector console, F12...then "Network", then visit the page, then click the red response, then click "Response" and paste that in)...we can maybe help. Otherwise we cannot.

**Comment by projecterik** [comment]
I am a friendly person.
I was born like that.
But keep in mind that i face the issue.
And yes i need help.
Because you are the builder.
So please were can i find what you are writing me?
Because again: you are using computer talk.

**Comment by SlvrEagle23** [comment]
@projecterik This is computer software. In order to fix computer software, we need computer talk. I can't tell you how to fix this without using computer language.

I gave you a specific set of instructions to follow in the previous message; if you are unwilling to follow those, we can't support you further.

**Comment by projecterik** [comment]
from the inspector console, F12 were can i find this in azuracast?

**Comment by SlvrEagle23** [comment]
@projecterik It's a feature in most browsers (if you're on Chrome, for example). Hitting the F12 key on your keyboard will open the inspector. From there, follow the other steps.

**Comment by projecterik** [comment]
![Uploading thumbnail_IMG-20210423-WA0000.jpg…]()

**Comment by SlvrEagle23** [comment]
@projecterik You hit comment before it finished. Let the upload finish before hitting comment.

**Comment by projecterik** [comment]
![thumbnail_IMG-20210423-WA0000](https://user-images.githubusercontent.com/80392954/115936785-7b8fbf80-a496-11eb-8d95-5a414f00cb9b.jpg)

**Comment by SlvrEagle23** [comment]
@projecterik Yep, this is the correct panel. Now click the "Network" tab, then go to the Music Files page. You should see one request in red, maybe with 500 next to it. Click that request, then click "Response" tab, then copy/paste that error message to us.

**Comment by projecterik** [comment]
Thank god it is the right age.
It is hard work for me you understand.
Were can i find network tab?

**Comment by SlvrEagle23** [comment]
@projecterik It's in that panel you just posted a screenshot of.

**Comment by projecterik** [comment]
![thumbnail_IMG-20210423-WA0000](https://user-images.githubusercontent.com/80392954/115937084-3d46d000-a497-11eb-9157-071f43515bbe.jpg)

**Comment by SlvrEagle23** [comment]
@projecterik Refresh the page and you should see requests pop up in that panel. Also something is wrong with your screenshots.

**Comment by projecterik** [comment]
/api/station/9/files…currentDirectory=:1 Failed to load resource: the server responded with a status of 500 ()
StationsMedia-9f7023c5f7.js:1 Uncaught (in promise) TypeError: Cannot read property 'message' of undefined
    at StationsMedia-9f7023c5f7.js:1
DevTools failed to load SourceMap: Could not load content for https://stream.zeeland-hosting.nl/static/dist/lib/bootstrap-vue/bootstrap-vue.min.css.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://stream.zeeland-hosting.nl/static/dist/lib/bootstrap/bootstrap.bundle.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://stream.zeeland-hosting.nl/static/dist/lib/bootstrap-vue/bootstrap-vue.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://stream.zeeland-hosting.nl/static/dist/lib/moment/moment.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://stream.zeeland-hosting.nl/static/dist/lib/moment/locales.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE

**Comment by SlvrEagle23** [comment]
@projecterik That's the "Console" tab; we need the actual network response from the "Network" tab.

**Comment by projecterik** [comment]
Cant find it i am sorry

**Comment by SlvrEagle23** [comment]
@projecterik You were just on it a second ago.

**Comment by projecterik** [comment]
Failed to load resource: the server responded with a status of 500 ()
StationsMedia-9f7023c5f7.js:1 Uncaught (in promise) TypeError: Cannot read property 'message' of undefined
    at StationsMedia-9f7023c5f7.js:1
DevTools failed to load SourceMap: Could not load content for https://stream.zeeland-hosting.nl/static/dist/lib/bootstrap/bootstrap.bundle.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://stream.zeeland-hosting.nl/static/dist/lib/bootstrap-vue/bootstrap-vue.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://stream.zeeland-hosting.nl/static/dist/lib/moment/moment.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://stream.zeeland-hosting.nl/static/dist/lib/moment/locales.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://stream.zeeland-hosting.nl/static/dist/lib/bootstrap-vue/bootstrap-vue.min.css.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE

**Comment by SlvrEagle23** [comment]
@projecterik That's the same thing you posted earlier. That's the wrong tab. You want the "Network" tab.

**Comment by projecterik** [comment]
I think this is what you need.

**Comment by projecterik** [comment]
![thumbnail_IMG-20210423-WA0000](https://user-images.githubusercontent.com/80392954/115937338-08874880-a498-11eb-8118-43298519317f.jpg)

**Comment by SlvrEagle23** [comment]
@projecterik Scroll up and down that list of requests...one will be red. Click it, then click "Response" in the little window that opens up. That should have an error.

**Comment by projecterik** [comment]
![thumbnail_IMG-20210423-WA0000](https://user-images.githubusercontent.com/80392954/115937443-59973c80-a498-11eb-9a1d-7c6e9bcf1be2.jpg)

**Comment by projecterik** [comment]
Request URL: https://stream.zeeland-hosting.nl/api/station/9/files/list?internal=true&rowCount=10&current=1&currentDirectory=
Request Method: GET
Status Code: 500 
Remote Address: 162.250.189.68:443
Referrer Policy: no-referrer-when-downgrade
content-type: text/html; charset=UTF-8
date: Fri, 23 Apr 2021 22:58:32 GMT
server: nginx
:authority: stream.zeeland-hosting.nl
:method: GET
:path: /api/station/9/files/list?internal=true&rowCount=10&current=1&currentDirectory=
:scheme: https
accept: application/json, text/plain, */*
accept-encoding: gzip, deflate, br
accept-language: nl-NL,nl;q=0.9,en-US;q=0.8,en;q=0.7
cookie: __tawkuuid=e::zeeland-hosting.nl::wuhNcYzwP9gipu9NC/zRP+y8HX1qMuf8jPHxi2pT0TWt4cbCmTtpnsDs+04kFRvf::2; cf_clearance=b061af1d8a6a93fd0b4be95ac9aa1cf62463b700-1618564581-0-150; __cfduid=dcc622455f5e86b927c3c4355350c11de1618564581; app_session=7e89e2045be3c55c9e092377079facd4
dnt: 1
referer: https://stream.zeeland-hosting.nl/station/9/files
sec-ch-ua: " Not A;Brand";v="99", "Chromium";v="90", "Google Chrome";v="90"
sec-ch-ua-mobile: ?0
sec-fetch-dest: empty
sec-fetch-mode: cors
sec-fetch-site: same-origin
user-agent: Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.85 Safari/537.36
internal: true
rowCount: 10
current: 1
currentDirectory:

**Comment by SlvrEagle23** [comment]
@projecterik Well...let's try another route.

In AzuraCast, go to System Administration (from the top right dropdown), then `Log Viewer`, then view the `AzuraCast Application Log` at the top. Paste in any errors you see there near the bottom.

**Comment by projecterik** [comment]
![thumbnail_IMG-20210423-WA0000](https://user-images.githubusercontent.com/80392954/115937744-dc1ffc00-a498-11eb-9e1a-c341dee022a5.jpg)

**Comment by SC2Mitch** [comment]
@projecterik I believe this will be very helpful for you, please review this clip I have created for you, it will show you how to provide us with this error, kind regards. 

![eee3g8IIMt](https://user-images.githubusercontent.com/29955942/115937806-b55dc780-a490-11eb-9351-b1d5185689db.gif)

**Comment by projecterik** [comment]
No that is not he;lpful at al.
I am not into computers i am a dj.
As i told before

**Comment by SlvrEagle23** [comment]
@projecterik I noticed from your screenshots you're also on the stable release. It's possible we fixed the issue you're experiencing in one of our Rolling Release updates since the latest stable release, so you may want to try switching release channels, which you can do by running, in SSH:

```
cd /var/azuracast
./docker.sh update-self
./docker.sh setup-release
```

Answer "N" when it asks whether you want to prefer stable releases, then run:

```
./docker.sh update
```

**Comment by projecterik** [comment]
Why is it not posible for you to do.
I am not a computer nerd.
I am not a programmer.
I am a dj.

**Comment by SlvrEagle23** [comment]
@projecterik Are you the owner/operator of the AzuraCast installation?

**Comment by projecterik** [comment]
why dont you du an updatefor me.
I dont have acces to the server.
Just make it work.
You are the programmmer not me.

**Comment by SlvrEagle23** [comment]
@projecterik We don't have access to your server either. We don't run your installation.

Are you the one who installed AzuraCast, or did someone else install it for you? If so, you should contact them for help in this case.

**Comment by projecterik** [comment]
You areresponsable.
You done the update.
Not me not us.
So why is the problem for me now.
Not fair!

**Comment by SlvrEagle23** [comment]
@projecterik We did not update your installation! We do not manage your AzuraCast installation! It is self-hosted software that you run on your own servers.

If you installed AzuraCast yourself, then you are responsible for updating it and maintaining it (and for knowing enough technical details to contact us with logs if something goes wrong).

Especially if, as is the case in this scenario, it's a reseller instance that is selling AzuraCast.

**Comment by projecterik** [comment]
The updates are comming from you.
And yes it is installed by one of us true.
But stil the updates are comming from you.
Andafter trhe update it is not working anymore.
So it is because of your stupid updates!

**Comment by SlvrEagle23** [comment]
@projecterik You should contact whoever installed your AzuraCast installation and provide them with contact details for us and instructions on how to update as we have provided in this thread.

We won't be providing any further support on this matter until we receive useful information we can use to diagnose the issue.

**Comment by projecterik** [comment]
I dont know if i have to thank you now.
This took a lot of my time.
And it didnt help me.

**Comment by SlvrEagle23** [comment]
@projecterik If it makes you feel any better, you also took up a lot of our time without helping us.
